src/components/MailchimpForm.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+
+import { css } from "@emotion/react";
+
+import { mediaQs } from "../mediaQueries";
+import { fonts, theme, copy  } from "../params";
+
+import addToMailchimp from 'gatsby-plugin-mailchimp';
+
+const MailchimpForm: React.FunctionComponent<{}> = () => {
+
+  const [email, setEmail] = React.useState('');
+
+  const [subscribed, setSub] = React.useState('');
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    addToMailchimp(email)
+      .then((data: any) => {
+        setSub('true');
+      })
+      .catch((error: Error) => {
+        // Errors in here are client side
+        // Mailchimp always returns a 200
+      });
+  };
+
+  const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.currentTarget.value);
+  };
+
+  return (
+    <form 
+      onSubmit={handleSubmit} 
+      className={"mc-form"}
+      css={css`
+        margin-bottom: 30px;
+      `}
+    >
+      <div>
+        <input
+          placeholder="Email address"
+          name="email"
+          type="text"
+          onChange={handleEmailChange}
+          css={css`
+            font-family: Myriad Pro;
+            display: inline-block;
+            max-width: 100%;
+            min-width: 300px;
+            height: 40px;
+            padding: 0 10px;
+            background: #fbf6ec;
+            border: none;
+            border-radius: 5px;
+            color:  #414C3B;
+            font-size: 20px;
+            font-weight: 400;
+          `}
+        />
+        {
+          !subscribed
+          ? 
+            <button 
+              type="submit"
+              css={css`
+                display: block;
+                width: 100%;
+                margin-top: 15px;
+                margin-left: 0;
+                color: #414C3B;
+                background: #D0AE23;
+                border: none;
+                font-size: 20px;
+                font-family: 'Myriad Pro';
```

Comment:
best to leverage the font variable here

see https://github.com/odeniso/higgins-site/blob/97f6ca61a344e909bd46b7145422d37fe3e3fe7f/src/components/TinyArticle.tsx#L18